extension/index.js

```diff
@@ -38,10 +42,19 @@ async function init() {
     addDependencies(createBox('Dev Dependencies'), devDependencies);
   }
 
-  if (dependencies.length && !pkg.private) {
-    const link = html`<a class="npmhub-anvaka btn btn-sm">Visualize full tree`;
-    link.href = `http://npm.anvaka.com/#/view/2d/${esc(pkg.name)}`;
-    dependenciesBox.insertBefore(link, dependenciesBox.firstChild);
+  if (!pkg.private) {
+    const realPkg = await window.backgroundFetch(getPkgUrl(pkg.name));
+    if (realPkg.name) { // if 404, realPkg === {}
+      const link = html`<a class="btn btn-sm">Open on npmjs.com`;
```

Comment:
Yep that's valid HTML :D 

npmhub has that unclosed tags forever. Also: https://github.com/npmhub/npmhub/blob/master/extension/index.js#L54-L55